python/lsst/afw/image/_exposureSummaryStats.py

```diff
@@ -117,7 +143,164 @@ def _read(bytes):
+        )
+        schema.addField(
+            "astromOffsetStd",
+            type="F",
+            doc="Standard deviation of offsets of astrometric calibration matches (arcsec)",
+        )
+        schema.addField("nPsfStar", type="I", doc="Number of stars used for PSF model")
+        schema.addField(
+            "psfStarDeltaE1Median",
+            type="F",
+            doc="Median E1 residual (starE1 - psfE1) for psf stars",
+        )
+        schema.addField(
+            "psfStarDeltaE2Median",
+            type="F",
+            doc="Median E2 residual (starE2 - psfE2) for psf stars",
+        )
+        schema.addField(
+            "psfStarDeltaE1Scatter",
+            type="F",
+            doc="Scatter (via MAD) of E1 residual (starE1 - psfE1) for psf stars",
+        )
+        schema.addField(
+            "psfStarDeltaE2Scatter",
+            type="F",
+            doc="Scatter (via MAD) of E2 residual (starE2 - psfE2) for psf stars",
+        )
+        schema.addField(
+            "psfStarDeltaSizeMedian",
+            type="F",
+            doc="Median size residual (starSize - psfSize) for psf stars (pixel)",
+        )
+        schema.addField(
+            "psfStarDeltaSizeScatter",
+            type="F",
+            doc="Scatter (via MAD) of size residual (starSize - psfSize) for psf stars (pixel)",
+        )
+        schema.addField(
+            "psfStarScaledDeltaSizeScatter",
+            type="F",
+            doc="Scatter (via MAD) of size residual scaled by median size squared",
+        )
+
+    def update_record(self, record: BaseRecord) -> None:
+        """Write summary-statistic columns into a record.
+
+        Parameters
+        ----------
+        record : `lsst.afw.table.BaseRecord`
+            Record to update.  This is expected to frequently be an
+            `ExposureRecord` instance (with higher-level code adding other
+            columns and objects), but this method can work with any record
+            type.
+        """
+        for field in dataclasses.fields(self):
+            value = getattr(self, field.name)
+            if field.name == "version":
+                continue
+            elif field.type.startswith("list"):
+                record[field.name][:] = value
+            else:
+                record[field.name] = value
+
+    @classmethod
+    def from_record(cls, record: BaseRecord) -> ExposureSummaryStats:
+        """Read summary-statistic columns from a record into ``self``.
+
+        Parameters
+        ----------
+        record : `lsst.afw.table.BaseRecord`
+            Record to read from.  This is expected to frequently be an
+            `ExposureRecord` instance (with higher-level code adding other
+            columns and objects), but this method can work with any record
+            type, ignoring any attributes or columns it doesn't recognize.
+
+        Returns
+        -------
+        summary : `ExposureSummaryStats`
```

Comment:
`lsst.afw.image.ExposureSummaryStats`?

Reply:
This one should be fine unqualified, as it's in the same module that defines this type.